packages/cli/build.config.ts

```diff
@@ -17,9 +18,22 @@ export default defineBuildConfig({
     "process.env.AGENTICA_VERSION": JSON.stringify(pkgJson.version), // replace version from package.json on build
   },
   rollup: {
-    inlineDependencies: true,
+    inlineDependencies: [
+      "typia",
+
+      "commander",
+
+      "picocolors",
+
+      // @clack/prompts related
+      "sisteransi",
+      "@clack/prompts",
+      "@clack/core",
+    ],
     esbuild: {
-      minify: true,
+      minify: !isCI, // minify only in CI and publish
+      target: "es2022", // support for Node.js 18
```

Comment:
node18 supports es2022

Reply:
if we can support node 18, we should decrease minimum engine version on package.json